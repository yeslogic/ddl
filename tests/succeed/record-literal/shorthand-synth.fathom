let x = false;
let y = true;
let z = false;
{x, y, z}