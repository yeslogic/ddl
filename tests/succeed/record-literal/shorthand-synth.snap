stdout = '''
let x : Bool = false; let y : Bool = true; let z : Bool = false; { x, y, z } : {
    x : Bool,
    y : Bool,
    z : Bool,
}
'''
stderr = ''
